pybossa/api/__init__.py

```diff
@@ -727,7 +727,8 @@ def get_service_request(task_id, service_name, major_version, minor_version):
         .get(authorized_services_key, [])
     )
     if service_name not in authorized_services:
-        return abort(403, "The project is not authorized to access this service")
+        authorized_services_403 = current_app.config.get("AUTHORIZED_SERVICES_403", "")
```

Comment:
It would be helpful to include a default error message here, including placeholders for project_id and service_name in case the config is ever missing the expected value. This way, users would still receive a reasonable error message.